src/main/webapp/script.js

```diff
@@ -700,11 +787,14 @@ function updateEventTagBox() {
  */
 async function getRecommendedEvents() {
   if (loggedIn) {
-    fetch('/user?get=saved').then((response) => response.json())
-        .then(function(js) {
+    getUserIDToken().then((userToken) => {
+      fetch('/user?get=saved&userToken=' + userToken)
+          .then((response) => response.json())
+          .then(function(js) {
         // TODO: change this fetch call to get recommendations instead
           getEvents(dummyEvents, 1, 0);
-        });
+      });
+    });
```

Comment:
Thanks for doing this. Please fix the lint errors

Reply:
The remaining lint errors are from calls to a variable declared in another script that is imported and for formatting that looks improper/less readable to me.

Reply:
Oh okay, yeah I was referring to the indentation lint errors. Thanks for explaining